Makefile

```diff
@@ -0,0 +1,17 @@
+box.phar:
+	curl -Lls https://github.com/humbug/box/releases/download/3.8.4/box.phar -o box.phar
+	chmod +x box.phar
+
+phpspec.phar: box.phar vendor
```

Comment:
I think it should also depend on the project source code, e.g. `$(shell find bin/ src/ -type f)` (see https://github.com/infection/infection/blob/master/Makefile#L208)